### ISSUE TITLE
Google play link broken - apk?

### ISSUE THREAD

**Issue by guysoft** [jas0nc/ANT2BLE_HeartRate#1]
Hey, 
Is there an APK? seems like google play link is broken

**Comment by meshaun9** [comment]
I agree  It appears to be broken. I navigated to the apk file in the repository tree and downloaded it, installed it and it opened and showed a white screen then closed. :/
